src/mudlet-lua/lua/LuaGlobal.lua

```diff
@@ -128,7 +128,8 @@ local packages = {
 -- TODO: extend to support common Lua code being placed in system shared directory
 -- tree as ought to happen for *nix install builds.
 local prefixes = {"../src/mudlet-lua/lua/", "../Resources/mudlet-lua/lua/",
-    "mudlet.app/Contents/Resources/mudlet-lua/lua/", "mudlet-lua/lua"}
+    "mudlet.app/Contents/Resources/mudlet-lua/lua/", "mudlet-lua/lua",
+    getMudletLuaDefaultPath()}
```

Comment:
Just to explain... The function returns a table, so the table is now {"path1", "path2", {"path from C", "another path from C"}}. The loop below iterates over the outer table only, you want both tables merged.

But the actual LuaGlobal is in#415